invenio_app_ils/circulation/config.py

```diff
@@ -56,8 +56,6 @@
 ILS_CIRCULATION_MAIL_MSG_CREATOR = (
     "invenio_app_ils.circulation.mail.factory:default_loan_message_creator"
 )
-#: Loan list message creator class
-ILS_CIRCULATION_LOANS_MAIL_MSG_CREATOR = "invenio_app_ils.circulation.mail.factory:default_loan_list_message_creator"
```

Comment:
@jrcastro2 please double check this clean up, it was all dead code after the moving of active loans email